nuget/spec/dependabot/nuget/nuget_client_spec.rb

```diff
@@ -0,0 +1,66 @@
+# typed: false
+# frozen_string_literal: true
+
+require "spec_helper"
+require "dependabot/nuget/nuget_client"
+
+RSpec.describe Dependabot::Nuget::NugetClient do
```

Comment:
Good to start this test class, I was relying on the using areas to test the functionality, but as it starts touching more spots that stops being as good of an idea.